packages/devtools_app/lib/src/inspector/flutter/inspector_screen_details_tab.dart

```diff
@@ -0,0 +1,229 @@
+    this.expandCollapseButtons,
+    this.expandCollapseSupported,
+    this.summaryTreeController,
+    Key key,
+  }) : super(key: key);
+
+  final bool expandCollapseSupported;
+  final Widget detailsTree;
+  final Widget expandCollapseButtons;
+  final InspectorTreeControllerFlutter summaryTreeController;
+
+  @override
+  Widget build(BuildContext context) {
+    final enableExperimentalStoryOfLayout =
+        InspectorTreeControllerFlutter.enableExperimentalStoryOfLayout;
+    final tabs = <Tab>[
+      const Tab(text: 'Details Tree'),
+      if (enableExperimentalStoryOfLayout) const Tab(text: 'Layout Details')
+    ];
+    final tabViews = <Widget>[
+      detailsTree,
+      if (enableExperimentalStoryOfLayout)
+        LayoutDetailsTab(controller: summaryTreeController),
+    ];
+    final focusColor = Theme.of(context).focusColor;
+    return Container(
+      decoration: BoxDecoration(
+        border: Border.all(color: focusColor),
+      ),
+      child: DefaultTabController(
+        length: tabs.length,
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: <Widget>[
+            Container(
+              padding: const EdgeInsets.only(bottom: 8.0),
+              child: Row(
+                children: <Widget>[
+                  Container(
+                    color: Theme.of(context).focusColor,
+                    child: TabBar(
+                      tabs: tabs,
+                      isScrollable: true,
+                    ),
+                  ),
+                  if (expandCollapseSupported)
+                    Expanded(
+                      child: expandCollapseButtons,
+                    ),
+                ],
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              ),
+            ),
+            Expanded(
+              child: TabBarView(
+                children: tabViews,
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class LayoutDetailsTab extends StatefulWidget {
+  const LayoutDetailsTab({Key key, this.controller}) : super(key: key);
+
+  final InspectorTreeControllerFlutter controller;
+
+  @override
+  _LayoutDetailsTabState createState() => _LayoutDetailsTabState();
+}
+
+class _LayoutDetailsTabState extends State<LayoutDetailsTab>
+    with AutomaticKeepAliveClientMixin<LayoutDetailsTab>
+    implements InspectorControllerClient {
+  InspectorTreeControllerFlutter get controller => widget.controller;
+
+  RemoteDiagnosticsNode get selected => controller.selection?.diagnostic;
+
+  // Lifecycle hooks
+  @override
+  void initState() {
+    super.initState();
+    controller.addClient(this);
+  }
+
+  @override
+  void dispose() {
+    super.dispose();
+    controller.removeClient(this);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    super.build(context);
+    if (selected == null)
+      return const Center(
+        child: CircularProgressIndicator(),
```

Comment:
don't show a progress indicator. Blank content is probably fine. A progress indicator would be appropriate if the inspector was still loading, not because nothing is yet selected.

Reply:
alright!